src/anm/analytics.js

```diff
@@ -19,23 +19,17 @@ var Analytics = function () {
         if (utils.is.not_empty(self.queue)) {
             var array = [];
             for (var key in self.queue) {
-                array.push(self.queue[key]);
+                if (self.queue.hasOwnProperty(key)) {
```

Comment:
Right, I shouldn't have used `forEach`. But here you say that it's ok. Are you sure that we don't need to support old browsers here?